README.md

```diff
@@ -0,0 +1,24 @@
+#### Our Project
+
+Web based postal client
+
+#### Current features
+
+- Connect using HTTPS
+- Create user
+- Connect to remote mail server and storage using SMTPS and IMAPS
+- Send simple message
+- Read and delete simple messages
+
+#### Future features
+
+- Web UI
+- JWT authentication
+- Correctly load complex email messages from remote storage
+- Create folders, set flags and move messages on remote mail storage
+- Send complex multipart messages with attachments
+- Better, practical REST api
+- Query support
+- Paging support
+- Multiple remote accounts per user
+- And more!
```

Comment:
There are signs showing that this file has new lines. But it all copies our current README.md, doesn't it?

Reply:
As I said, let's start the discussion. I've made no changes to readme or anything else yet.

Reply:
Well, my fault